mint.json

```diff
@@ -113,12 +112,11 @@
           "group": "Diffusion Models",
           "pages": [
             "cerebrium/fine-tuning/diffusion-models/introduction",
-            "cerebrium/fine-tuning/diffusion-models/dataset",
             "cerebrium/fine-tuning/diffusion-models/training",
+            "cerebrium/fine-tuning/diffusion-models/dataset",
             "cerebrium/fine-tuning/diffusion-models/config"
           ]
-        },
-        "cerebrium/fine-tuning/init-training-project"
```

Comment:
why remove?

Reply:
I moved it into the text instead of a seperate file